test/__fixtures__/async/transformed/spec.js

```diff
@@ -0,0 +1,39 @@
+const Login = require('./page')
+
+describe('Test describe', () => {
+    before(async () => {
+        await Authenticate.login()
+    })
+
+    it('testing general commands', async () => {
+        await $('.foo').click()
+        await browser.pause(5000)
+        await browser.waitUntil(async () => {
+            return (await $('.bar').getText()) === 'foo';
+        })
+
+        const arr = await $$('.foo')
+
+        await browser.execute(() => {
+            $('.foo').click();
+        });
+
+        await browser.execute(() => {
+            document.querySelector('.foo').click();
+            $('.foo').click();
+        });
+    })
+
+    it('testing loops', async () => {
+        for (const num of [1, 2, 3]) {
+            await $('.foo').setValue(num)
+        }[(1, 2, 3)].map(async num => {
```

Comment:
I found an issue here I need to look into.

Reply:
`[(1, 2, 3)].map`

That is automatically being added by the parser when there are no transforms applied and the expression above it doesn't have an ending semi colon.